### PR TITLE
Gojsonnet

### PR DESCRIPTION
**Pull request by janeklb**
This is just a copy of #728, rebased from master's HEAD.

Creating a new PR just to see if tests pass in travis-ci

### CODE REVIEW

**Comment by janeklb** [comment]
🤔  travis-ci integration seems to be broken -- last few PRs haven't triggered builds either

**Comment by ramaro** [comment]
@janeklb thanks for this. Yes, travis is broken at the moment and we're aware and working on it and will get back to you soon.

File: kapitan/targets.py [hunk 1]
```diff
@@ -86,7 +84,7 @@ def compile_targets(
                 logger.info("No changes since last compilation.")
                 return
 
-    pool = multiprocessing.Pool(parallel)
+    pool = multiprocessing.get_context("spawn").Pool(parallel)
```
Comment (pvanderlinden) [inline kapitan/targets.py:87]: It's better to change it here: https://github.com/kapicorp/kapitan/blob/1c1a00b5266e9cc57c95e4430240f325af9ec088/kapitan/cli.py#L102

**Review by pvanderlinden — CHANGES_REQUESTED** [review]
Thanks @janeklb I added a comment to fix a mistake I made in the original PR.

File: kapitan/targets.py (hunk 1 above)
Comment (janeklb) [reply kapitan/targets.py]: Thanks @pvanderlinden - updated as requested

**Review by pvanderlinden — APPROVED** [review]

**Comment by ramaro** [comment]
@janeklb @pvanderlinden thanks both! LGTM, merging.

**Pull request merged**